modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/test/java/com/liferay/dynamic/data/mapping/internal/upgrade/v1_0_0/DynamicDataMappingUpgradeProcessTest.java

```diff
@@ -68,61 +49,48 @@
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.HashSet;
-import java.util.LinkedHashSet;
 import java.util.List;
 import java.util.Locale;
 import java.util.Map;
 import java.util.Set;
 
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
 import org.junit.Test;
-import org.junit.runner.RunWith;
-
-import org.mockito.Matchers;
-import org.mockito.Mock;
-import org.mockito.invocation.InvocationOnMock;
-import org.mockito.stubbing.Answer;
-
-import org.powermock.api.mockito.PowerMockito;
-import org.powermock.core.classloader.annotations.PowerMockIgnore;
-import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.core.classloader.annotations.SuppressStaticInitializationFor;
-import org.powermock.modules.junit4.PowerMockRunner;
 
 import org.skyscreamer.jsonassert.JSONAssert;
 
 /**
  * @author Marcellus Tavares
  */
-@PowerMockIgnore("javax.xml.stream.*")
-@PrepareForTest({LocaleUtil.class, PropsValues.class})
-@RunWith(PowerMockRunner.class)
-@SuppressStaticInitializationFor(
-	{
-		"com.liferay.portal.kernel.xml.SAXReaderUtil",
-		"com.liferay.portal.util.PropsValues"
-	}
-)
-public class DynamicDataMappingUpgradeProcessTest extends PowerMockito {
+public class DynamicDataMappingUpgradeProcessTest extends BaseDDMTestCase {
+
+	@ClassRule
+	@Rule
+	public static final LiferayUnitTestRule liferayUnitTestRule =
```

Comment:
Is there any particular need for this rule to be declared?

Reply:
No, I only included it because SF asked me to 😅  

<img width="1661" alt="Captura de Tela 2022-02-04 às 13 15 50" src="https://user-images.githubusercontent.com/34186632/152563750-7cf45dea-bf0a-4e06-b4ba-fc3d81f9d37d.png">

Reply:
🤷🏻 
I guess it doesn't take into account the heritage.